src/App.js

```diff
@@ -1,3 +1,4 @@
 import Calculator from './components/Calculator';
+import Quote from './components/Quote';
 
-export default Calculator;
+export default { Calculator, Quote };
```

Comment:
- [ ] Instead of using this file as an index, I would suggest that you create an actual component that wraps your entire app. By convention, an `<App />` component is used at the top of your component tree. 

```js
// App.js
import Calculator from './components/Calculator';
import Quote from './components/Quote';

const App = () => {
  return (
    <>
      <Calculator />
      <Quote />
    </>
  )
}

export default App;
```


```js
// index.js
import React from 'react';
import ReactDOM from 'react-dom/client';
import App from './App';
import reportWebVitals from './reportWebVitals';

const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
);

```

Please consider applying this! 🤓 🔧